scripts/run_test262_suite.py

```
@@ -518,330 +521,474 @@ def evaluate_suite(
+                    runner_timeout = int(os.environ.get(
+                        "TEST262_RUNNER_TIMEOUT_S",
+                        str(runner_timeout_default),
+                    ))
+                    captured_stderr_chunks: list[str] = []
+                    process: subprocess.Popen | None = None
+                    stderr_thread: threading.Thread | None = None
+
+                    def _flush_runner_state() -> None:
+                        nonlocal runner_output
+                        runner_output = (
+                            "".join(captured_stderr_chunks).strip())
```

Ruff (0.15.12)

[warning] Function definition does not bind loop variable `captured_stderr_chunks`

(B023)